common/src/main/java/com/example/android/uamp/media/library/BrowseTree.kt

```diff
@@ -147,7 +154,10 @@ const val UAMP_BROWSABLE_ROOT = "/"
 const val UAMP_EMPTY_ROOT = "@empty@"
 const val UAMP_RECOMMENDED_ROOT = "__RECOMMENDED__"
 const val UAMP_ALBUMS_ROOT = "__ALBUMS__"
+const val UAMP_RECENT_ROOT = "//"
```

Comment:
Perhaps `__RECENT__` would be more descriptive and useful when debugging

Reply:
Done!